package.json

```diff
@@ -34,5 +37,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
Move these libraries to dependencies.
    "@reduxjs/toolkit": "^2.2.3",
    "axios": "^1.6.8",
    "react-redux": "^9.1.0",